### PR TITLE
VS-757 - Use JASIX to make sub-jsons of annotated output of Nirvana

### PR DESCRIPTION
**Pull request by gbggrant**
This PR uses Nirvana's JASIX parser to break the annotated output of Nirvana into a genes and positions json which are then parsed by the corresponding python methods. This improves performance significantly.

Successful workflow here:
https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/14d2d1cf-3a2a-42e0-8542-c70edd2133e5

### CODE REVIEW

File: .dockstore.yml [hunk 1]
```diff
@@ -142,6 +142,7 @@ workflows:
          - master
          - ah_var_store
          - rc-vs-651-vat-from-vds
+         - gg_VS-757_JASIX_make_subjsons
```
Comment (gbggrant) [inline .dockstore.yml:145]: Remove this prior to merge

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 2]
```diff
@@ -176,7 +174,7 @@ task MakeSubpopulationFilesAndReadSchemaFiles {
     # ------------------------------------------------
     # Runtime settings:
     runtime {
-        docker: "us.gcr.io/broad-dsde-methods/variantstore:VS-561_var_store_2022_12_08"
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:gg_VS-757_var_store_2022_12_15d"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:177]: Replace this with a properly named docker prior to merge

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 3]
```diff
@@ -221,7 +219,7 @@ task StripCustomAnnotationsFromSitesOnlyVCF {
...
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:222]: Replace this with a properly named docker prior to merge

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 4]
```diff
@@ -392,15 +403,15 @@ task AnnotateVCF {
     # ------------------------------------------------
     # Outputs:
     output {
-        File annotation_json = "~{annotation_json_name}"
-        File annotation_json_jsi = "~{annotation_json_name_jsi}"
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:396]: No longer providing nor saving the full annotation jsons

File: scripts/variantstore/wdl/extract/create_genes_bqloadjson_from_annotations.py [hunk 5]
```diff
@@ -46,6 +49,9 @@ def make_genes_json(annotated_json, output_genes_json):
     output_genes_file.close()
     json_data.close()
 
+    if gene_count == 0:
+        print(f"ERROR - Found no items in annotated json file: {annotated_json}")
```
Comment (koncheto-broad) [inline scripts/variantstore/wdl/extract/create_genes_bqloadjson_from_annotations.py:53]: nit: why not logging.error for consistency?

File: scripts/variantstore/wdl/extract/create_vt_bqloadjson_from_annotations.py [hunk 6]
```diff
@@ -331,6 +329,9 @@ def make_positions_json(annotated_json, output_json):
     output_file.close()
     json_data.close()
 
+    if position_count == 0:
+        print(f"ERROR - Found no items in annotated json file: {annotated_json}")
```
Comment (koncheto-broad) [inline scripts/variantstore/wdl/extract/create_vt_bqloadjson_from_annotations.py:333]: nit: why not logging.error for consistency?

**Comment by koncheto-broad** [comment]
LGTM!

Have any numbers on the performance improvement?  It wasn't easy to see an old run before Jasix that would be a good 1-to-1 comparison.

**Review by koncheto-broad — APPROVED** [review]

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 7]
```diff
@@ -265,7 +263,6 @@ task RemoveDuplicatesFromSitesOnlyVCF {
         bcftools view --threads 4 -i 'REF~"N"' -O u sites_only.bcf | bcftools query -f '%CHROM\t%POS\t%ID\t%REF\t%ALT\n' > track_dropped.tsv
 
         echo_date "VAT: filter out sites with N's in the reference AND sites with AC=0"
-        # TODO - NOTE - we are NOT tracking the sites with AC=0 - should we? For Lee (Rori is checking)
         bcftools view --threads 4 -e 'REF~"N" || AC=0' -O b sites_only.bcf -o filtered_sites_only.bcf
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:266]: Add a comment directing users how to find the AC=0 sites.

**Pull request merged**